### PR TITLE
Rounding Numeric of Values

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://jira.rax.io/browse/MNRVA-807

**Description:**

1) Created **roundUnitsValue** function for rounding decimal values, bytes and values multiple of 1000 to 1k.
2) pass value from data-points, x-axis and y-axis to this function to get formatted value.

**Screenshot:**

Screenshot 2021-07-14 at 6 14 13 PM

### CODE REVIEW

File: src/line-graph.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ export class LineGraph extends HTMLElement {
     // Create the lines
     var line = d3.line()
       .x(d => xScale(d.time))
-      .y(d => yScale(d.value));
+      .y(d => yScale(Utils.roundUnitsValue('percent', d.value)));
```
Comment (FarajiA) [inline src/line-graph.js:140]: The hard coded value will be changed.

**Comment by FarajiA** [comment]
A possible scenario would be bytes or megabyte values needing rounding.

**Review by FarajiA — APPROVED** [review]

**Comment by lovedeep92** [comment]
@FarajiA  Taken care the scenario!

File: src/core/utils.js [hunk 2]
```diff
@@ -63,4 +66,59 @@ export class Utils {
         });
         return arr;
     }
+        /**
+         * Check byte unit of measurment
+         * @param {number} n
+         * @param {number} d
+         * @returns number
+         */
+         static roundOffValue(n, d=2) {
```
Comment (pras7430) [inline src/core/utils.js:75]: I think we can change its indentation and make it a little descriptive

**Review by pras7430 — APPROVED** [review]

File: src/core/utils.js (hunk 2 above)
Comment (lovedeep92) [reply src/core/utils.js:75]: sure!!

File: src/core/utils.js [hunk 3]
```diff
@@ -63,4 +63,59 @@ export class Utils {
...
+            switch(measurmentUnit) {
+                case measurmentUnit === 'bytes':
+                case measurmentUnit === 'kilobytes':
+                case measurmentUnit ===  'megabytes':
+                    return this.formatBytes(value, 2);
+                case measurmentUnit === 'load':
+                case measurmentUnit === 'count':
+                    return this.roundOffValue(value);
+                case measurmentUnit === 'frames':
+                    return this.roundOffValue(value) + ' frames/s';
```
[the remark this answers is not in the document]
Comment (lovedeep92) [reply src/core/utils.js:96]: got it!

[the remark this answers is not in the document]
Comment (lovedeep92) [reply src/core/utils.js:96]: if it so, then how we will append different units with value?

File: src/line-graph.js (hunk 1 above)
Comment (lovedeep92) [reply src/line-graph.js]: done!

File: src/core/utils.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (lovedeep92) [reply src/core/utils.js]: done!!

File: src/core/utils.js (hunk 2 above)
Comment (lovedeep92) [reply src/core/utils.js]: modified!!

**Pull request merged**